olp-cpp-sdk-authentication/include/olp/authentication/Settings.h

```diff
@@ -55,6 +55,9 @@ struct AUTHENTICATION_API Settings {
   /// Client credentials
   AuthenticationCredentials credentials;
 
+  /// Optional. Scope to authorize to.
+  boost::optional<std::string> scope;
```

Comment:
provide default value

```suggestion
  boost::optional<std::string> scope{boost::none};
```